docs/content/policy-language.md

```diff
@@ -1517,6 +1517,78 @@ q {
 ```live:eg/assignment2:output:expect_assigned_above,expect_referenced_above
 ```
 
+#### Membership `in`
```

Comment:
I guess it could be useful to see some examples together with `not`, as that solves a real problem many currently face. Something like:

```rego
not "admin" in input.user.roles
```